general_rules/Procedure.tex

```diff
@@ -181,13 +181,13 @@ \subsection{Operators}
 
 \subsection{Moderator}
 \label{rule:moderator}
-The LOC is responsible of organizing test moderation in the local language.
+The LOC is responsible for organizing test moderation in the local language.
 The OC may request the participating teams to provide a team member for moderation.
 Candidates have to be fluent in the moderation language (default is English).
 
 \noindent\textbf{Responsibilities:} The moderators have to:
 \begin{compactitem}
-	\item Do \textbf{NOT interfere} with the performance
+	\item \textbf{NOT interfere} with the performance
```

Comment:
*Do NOT* is more correct. Emphasizes

Reply:
I was thinking in context of the full sentence here:
"The moderators have to NOT interfere..."

For the rest, let's wait for a native speaker.

Reply:
Yep, I got your intention, but as when you have this signs with 2 words you write:

- No Smoking
- Do NOT smoke in this area

The *do* adds three tons of stress. On the rest, I agree in having an expert in English Grammar =P, I often make stupid mistakes (e.g. with in/on/at which all map to *en* in Spanish).

Reply:
You don’t need the “do” if you want to add stress you can replace “have to” with “must”

Reply:
The native has spoken. Resolving.